src/main/java/seedu/address/logic/commands/AddAppointmentCommand.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+
+import java.util.Calendar;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Appointment.Appointment;
+import seedu.address.model.person.ReadOnlyPerson;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+//@@author chenxing1992
+
+/**
+ * Command to add Appointment to a person in addressBook
+ */
+public class AddAppointmentCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "Appointment";
+    public static final String COMMAND_ALIAS = "appt";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an Appointment to a person in address book. \n"
+            + COMMAND_ALIAS + ": Shorthand equivalent for add. \n"
+            + "Parameters: " + COMMAND_WORD + " INDEX "
+            + PREFIX_DATE + "DESCRIPTION, TIME" + "\n"
+            + "Example 1:" + COMMAND_WORD + " 1 "
+            + PREFIX_DATE + "Lunch, Next Monday 3pm";
+
+    public static final String MESSAGE_SUCCESS = "New Appointment added. ";
+    public static final String INVALID_PERSON = "This person is not in your address book";
+    public static final String INVALID_DATE = "Invalid Date. Please enter a valid date.";
+    public static final String SORT_APPOINTMENT_FEEDBACK = "Rearranged contacts to show upcoming appointments.";
+
+
+    private final Index index;
+    private final Appointment appointment;
+
+    public AddAppointmentCommand(Index index, Appointment appointment) {
+        this.index = index;
+        this.appointment = appointment;
+    }
+    //@@author chenxing1992
```

Comment:
no need to add //@@author to every method, just at the beginning of the class can alr